frontend/src/common/types/characteristics-list/characteristics-list.ts

```diff
@@ -0,0 +1,16 @@
+export type CharacteristicsListInfoProps = {
+  modelName: string;
+  importantFeatures: string[];
+  optionGroups: CharacteristicsGroupProps[];
+};
+
+export type CharacteristicsGroupProps = {
+  name: string;
+  options: CharacteristicsOption[];
+};
+
+export type CharacteristicsOption = {
+  value: string;
+  name?: string;
+  color?: string;
+};
```

Comment:
I have a few questions for this type:
1. What will each field contain if they all are defined?
2. Why do we have a separate field for the color? Isn't it just one of the options

Reply:
<img width="802" alt="example2" src="https://user-images.githubusercontent.com/106184656/186773923-0e894de0-0928-4abe-bd96-e436f2d7bd69.png">

Reply:
Would I should add one component with its own type (value, name, color) for the first list and a second component with the type (only value) for other lists?

Reply:
Okay, after the screenshot it started to make sense. Could you please attack a link to it in the comment for the `CharacteristicsOption`?